.github/workflows/build-envoy-images-release.yaml

```diff
@@ -210,3 +212,4 @@ jobs:
         run: |
           echo "Digests:"
           echo "quay.io/${{ github.repository_owner }}/cilium-envoy:${{ github.sha }}@${{ steps.docker_build_cd.outputs.digest }}"
+          echo "quay.io/${{ github.repository_owner }}/cilium-envoy:${{ env.ENVOY_MINOR_RELEASE }}-${{ github.sha }}"
```

Comment:
Should print out the "@<digest>" also for the new line so the full reference is easy to cut&paste?